src/mumble/mumble_pch.hpp

```diff
@@ -7,6 +7,8 @@
 #ifndef MUMBLE_MUMBLE_MUMBLE_PCH_H_
 #define MUMBLE_MUMBLE_MUMBLE_PCH_H_
 
+#include <cmath>
```

Comment:
I think that it's better if we include it after or just before `<math.h>`.

Reply:
Please, if you think something, please state why you think it. Make it actionable. Or, if it's just an opinion, or you think it's cleaner, say that. So: why do you think putting it there is better?

I put cmath here because I want standard library files to be at the top of mumble_pch.hpp. Why? From the perspective that standard library headers are lower-level than the high level library includes we have later on in the file.

With that in mind, you're right that math.h (and possibly others) also belong at the beginning of the file, but I didn't want to shake things up too much in this PR.

There are various things in mumble_pch.hpp that could use a cleanup.

Reply:
It's just because including it at the top of the file looks "out of place" to me, as there are a lot of defines below it, before other headers are included.

It's not a problem at all though, as we can reorder the headers in a dedicated PR, as you said.

Reply:
Ah, I can see what you're getting at.
Yes, this change to just add this lone header at the top of mumble_pch.hpp does look odd.

However, there's a method to the madness: the plan is to put libc headers first, then STL headers, then OS headers (Win32? CoreFoundation/Foundation/Appkit? etc.), then library headers.

Reply:
Sounds good!